3rdparty/python/requirements.txt

```diff
@@ -3,6 +3,10 @@ beautifulsoup4>=4.6.0,<4.7
 fasteners==0.15.0
 freezegun==1.0.0
 
+# Note: we use humbug to report telemetry. When upgrading, ensure the new version maintains the
+# anonymity promise we make here: https://www.pantsbuild.org/docs/anonymous-telemetry
```

Comment:
<img width="1068" alt="Screen Shot 2021-03-14 at 10 44 10 PM" src="https://user-images.githubusercontent.com/1268088/111108818-d1c42700-8516-11eb-9c7e-620c5920a2a7.png">

Reply:
nvm found it under https://www.pantsbuild.org/v2.4/docs/anonymous-telemetry